README.md

```diff
@@ -3,39 +3,32 @@
 [![Coverage Status](https://codecov.io/gh/Azure/aks-engine/branch/master/graph/badge.svg)](https://codecov.io/gh/Azure/aks-engine)
 [![GoDoc](https://godoc.org/github.com/Azure/aks-engine?status.svg)](https://godoc.org/github.com/Azure/aks-engine)
 
-## Overview
-
-AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and implements cluster provisioning and lifecycle operations for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
-
-More info, including a thorough walkthrough is [here](docs/aksengine.md).
+AKS-Engine is the easiest way to provision a DIY Kubernetes cluster on Azure.
```

Comment:
I still see the "DIY" in the diff. Perhaps commits squashing got too cute?

Reply:
Yep... Attempt 2 coming up